index.html

```diff
@@ -0,0 +1,321 @@
+        <a href="#"><img src="./images/logo.png" alt="logo" /></a>
+        <ul class="sid-cons">
+          <li>Portoflio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
+    <div class="headline-container">
+      <div class="items1">
+        <h1 class="items item1">I'm Gasangwa Glad to see you !</h1>
+        <p class="items item3">
+          I am a software developer. I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need to be coded, don't hesitate to
+          contact me.
+        </p>
+        <div>
+          <h4 class="contact">LET'S CONNECT</h4>
+          <ul class="icons">
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/twitter.png"
+                  alt="twitter-icon"
+                  class="twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/linkedin.png"
+                  alt="linkedin-icon"
+                  class="linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/medium.png"
+                  alt="medium-icon"
+                  class="medium"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/github.png"
+                  alt="github-icon"
+                  class="github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./images/angellist.png"
+                  alt="angellist-icon"
+                  class="angellist"
+              /></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    </div>
+    <!--html of the middle section-->
+    <section class="middle-section">
+      <div class="card1">
+        <div class="snapshoot">
+          <img src="./images/screenshot1.png" alt="screenshot" class="responsive"/>
+        </div>
+        <div class="word-title">
+          <div class="w-title">
+            <h2 class="name">Thomas</h2>
+            <ul class="canop">
+              <li>CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <div class="p-text">
+            <p class="word">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div>
+            <ul class="languages">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <img src="./images/see-project.png" alt="button" class="tap">
+        </div>
+      </div>
```

Comment:
- [ ] You have used an image in place of a `<button>` tag for the "See Project" button. Please remove the image and use a button tag instead. Do this for all your images.
- [ ] Also kindly implement the button default, hover and active states as shown in the Figma design below 👇 
![image](https://user-images.githubusercontent.com/46839548/166105675-ca7e6c0d-54bf-4cb3-ad34-5f073ab046d1.png)